models/protocol_tests/generate.go

```diff
@@ -23,6 +23,7 @@ type testSuite struct {
 	*api.API
 	Description string
 	Cases       []testCase
+	Input       bool
```

Comment:
Would be nice if `Input` was a little more descriptive. What do you think of something like a `Type` field. This would be more extensible.

```go
type TestSuiteType int
const (
    TestSuiteTypeInput = iota
    TestSuiteTypeOutput
    // Could add other categories later. 
)

type TestSuite struct {
    //...
    Type TestSuiteType
}
```